sdk/identity/Azure.Identity/tests/Mock/MockIdentityClient.cs

```diff
@@ -1,4 +1,5 @@
-﻿using System;
+﻿using Azure.Core;
```

Comment:
nit:  Should the copyright blurb be in these?  Doesn't look like the new files in the test area have it.